Source/Scene/TimeDynamicPointCloud.js

```diff
@@ -0,0 +1,554 @@
+
+        /**
+         * Options for controlling point size based on geometric error and eye dome lighting.
+         * @type {PointCloudShading}
+         */
+        this.pointCloudShading = new PointCloudShading(options.pointCloudShading);
+
+        /**
+         * The style, defined using the
+         * {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language},
+         * applied to each point in the point cloud.
+         * <p>
+         * Assign <code>undefined</code> to remove the style, which will restore the visual
+         * appearance of the point cloud to its default when no style was applied.
+         * </p>
+         *
+         * @type {Cesium3DTileStyle}
+         *
+         * @example
+         * pointCloud.style = new Cesium.Cesium3DTileStyle({
+         *    color : {
+         *        conditions : [
+         *            ['${Classification} === 0', 'color("purple", 0.5)'],
+         *            ['${Classification} === 1', 'color("red")'],
+         *            ['true', '${COLOR}']
+         *        ]
+         *    },
+         *    show : '${Classification} !== 2'
+         * });
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+         */
+        this.style = options.style;
+
+        this._clock = options.clock;
+        this._intervals = options.intervals;
+        this._clippingPlanes = options.clippingPlanes;
+        this._pointCloudEyeDomeLighting = new PointCloudEyeDomeLighting();
+        this._loadTimestamp = undefined;
+        this._clippingPlanesState = 0;
+        this._styleDirty = false;
+        this._pickId = undefined;
+        this._totalMemoryUsageInBytes = 0;
+        this._frames = [];
+        this._previousInterval = undefined;
+        this._nextInterval = undefined;
+        this._lastRenderedFrame = undefined;
+        this._clockMultiplier = 0.0;
+        this._runningLoadTime = 0.0;
+        this._runningLoadedFramesLength = 0;
+    }
+
+    defineProperties(TimeDynamicPointCloud.prototype, {
+        /**
+         * The {@link ClippingPlaneCollection} used to selectively disable rendering the point cloud.
+         *
+         * @memberof TimeDynamicPointCloud.prototype
+         *
+         * @type {ClippingPlaneCollection}
+         */
+        clippingPlanes : {
+            get : function() {
+                return this._clippingPlanes;
+            },
+            set : function(value) {
+                ClippingPlaneCollection.setOwner(value, this, '_clippingPlanes');
+            }
+        },
+
+        /**
+         * The total amount of GPU memory in bytes used by the point cloud.
+         *
+         * @memberof TimeDynamicPointCloud.prototype
+         *
+         * @type {Number}
+         * @readonly
+         *
+         * @see TimeDynamicPointCloud#maximumMemoryUsage
+         */
+        totalMemoryUsageInBytes : {
```

Comment:
Are you sure that some of these properties should be in MB and others in bytes?  What does the rest of Cesium do?

Reply:
I copied what `Cesium3DTileset` does. I think the premise is any option that is settable from the user is in MB and anything internal is bytes.

Reply:
Perhaps submit an issue to add this to the Coding Guide?  Or just update it.